<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncreaseMemAlloc</class>
 <widget class="QDialog" name="IncreaseMemAlloc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Change the amount of memory Disgaea PC should allocate when loading the game data.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Disgaea PC, unlike more modern games, doesn't allocate its memory dynamically based on the size of the game files. Instead it has a fixed size buffer which is created when the game is started. If you mod the game with files which exceed the size of buffer, then the game just crashes. This utility allows you to modify the Disgaea PC .exe to change the size of that buffer to accommodate bigger file sizes. (Thx to Kelebek1) </string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Step 1: Locate Disgaea PC exe</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="pathEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenExe">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Step 2: Change size of memory allocation</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="spMemSize">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>2000000000</number>
         </property>
         <property name="singleStep">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>byte</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>HEX preview:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="preview">
       <property name="font">
        <font>
         <family>Prompt</family>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Please locate Disgaea PC exe</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOverride">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save As</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
